source/fundamentals/connection/connect.txt

```diff
@@ -77,12 +77,36 @@ In the example, we set two connection options: ``maxPoolSize=20`` and
 
 .. _connect-atlas-java-driver:
 
-The following code shows how you can use the sample connection URI in a client
-to connect to MongoDB.
+Atlas Connection Example
+------------------------
+
+To connect to a MongoDB deployment on Atlas, create a client. You can
+create a client that uses your connection string and other 
+client options by passing a ``MongoClientSettings`` object to the
+``MongoClients.create()`` method.
+
+To create a ``MongoClientSettings`` object, use the
+``MongoClientSettings.builder()`` method.
+
+To specify your connection URI, pass it to
+the ``ConnectionString`` constructor, pass the constructor to the
+``applyConnectionString()`` method, and chain this method to the
+``MongoClientSettings.builder()`` method. To specify any other client
+options, chain the relevant methods to the
+``MongoClientSettings.builder()`` method.
+
+You can set the {+stable-api+} version as a client option to avoid
```

Comment:
Suggestion:

Since there's no object specified as to what the reader is setting the client option on, seems like "as a" is unnecessary. Suggested wording matches the usage in line 104 of this PR.

```suggestion
You can set the {+stable-api+} version client option to avoid
```